src/System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -124,6 +140,28 @@ private string GetPathWithCorrectDirectoryCapitalization(string fullPath)
             return fullPath.TrimSlashes();
         }
 
+        /// <inheritdoc />
+        public MockFileData AdjustTimes(MockFileData fileData, TimeAdjustments timeAdjustments)
+        {
+            var now = (dateTimeProvider ?? defaultDateTimeProvider)();
+            if ((timeAdjustments & TimeAdjustments.CreationTime) != TimeAdjustments.None)
```

Comment:
Not wrong, but I'd find the following more idiomatic
```suggestion
            if ((timeAdjustments & TimeAdjustments.CreationTime) != TimeAdjustments.CreationTime)
```

Reply:
```suggestion
            if (timeAdjustments.HasFlag(TimeAdjustments.CreationTime))
```

Reply:
Will replace with `HasFlag` in next update